### PR TITLE
Make vocab

### PR DESCRIPTION
**Pull request by LiyuanLucasLiu**
the original implementation of ```make_vocab``` does not support vocabulary configuration (like min_count), also, the comments (and doc) miss the ```--include-package INCLUDE_PACKAGE```

### CODE REVIEW

**Comment by joelgrus** [comment]
I think this looks good, but can you add a simple test for the new functionality? Thanks!

**Comment by LiyuanLucasLiu** [comment]
I've modified the original test case for ```make_vocab```, but i'm not sure how to run this test...

**Comment by joelgrus** [comment]
assuming you have pytest installed, you can run just that with

```
pytest -v -k TestMakeVocab
```

the -k only runs test classes that match what you pass it.

--

also, can you leave the original test as is (so that we still test that case) and just create a separate one for passing in the params? thanks

**Comment by LiyuanLucasLiu** [comment]
this pytest is cool! I've created a new test case for the config :-)

File: allennlp/commands/make_vocab.py [hunk 1]
```diff
@@ -70,8 +72,8 @@ def make_vocab_from_params(params: Params):
     prepare_environment(params)
 
     vocab_params = params.pop("vocabulary", {})
-    vocab_dir = vocab_params.get('directory_path')
-    if vocab_dir is None:
+    vocab_dir = vocab_params.pop('directory_path', {})
```
Comment (joelgrus) [inline allennlp/commands/make_vocab.py]: one last nit: can you change the default value to `None`? (`{}` doesn't make sense as a possible value)

**Comment by LiyuanLucasLiu** [comment]
ok, no problem

**Pull request merged**